File: backend/src/main/java/com/team05/petmeeting/domain/naming/dto/BadWordAddReq.java
package com.team05.petmeeting.domain.naming.dto;

import jakarta.validation.constraints.NotBlank;

// 금칙어 추가 요청 DTO
public record BadWordAddReq(
        @NotBlank(message = "추가할 금칙어를 입력해주세요.")
        String badWord
) {}

File: backend/src/main/java/com/team05/petmeeting/domain/naming/dto/BadWordAddReq.kt
package com.team05.petmeeting.domain.naming.dto

import jakarta.validation.constraints.NotBlank

// 금칙어 추가 요청 DTO
data class BadWordAddReq(
        @field:NotBlank(message = "추가할 금칙어를 입력해주세요.")
        val badWord: String
) 